modelscope_agent/__init__.py

```diff
@@ -1 +1,48 @@
+import ray
```

Comment:
move this to _create_remote to avoid depends on ray when run it locally

Reply:
removed